_posts/2020-09-04-meet-the-team.md

```diff
@@ -0,0 +1,139 @@
+
+### What made you join the Solidity team?
+
+*Working on an open-source project for the first time was very motivating for me, especially combined with such an exciting and high potential environment like the blockchain space.*
+
+## Franzi - [@franzihei](https://github.com/franzihei)
+
+### What's your professional background?
+
+*After studying International Business Administration, I joined the innogy Innovation department working as a project manager and later portfolio manager with a focus on distributed technologies & blockchain (and what impact they can have on the energy sector). As part of that, I worked with several energy x blockchain projects like Co-tricity and Share&Charge. Following that, I became Head of Communications at brainbot, a blockchain development studio, after which I decided to join the Solidity team. :)*
+
+### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+*I'm supporting the team with communications and project management. This includes maintaining all communication channels like the @solidity_lang Twitter and the Solidity blog, organizing conferences and hackathons like this year's Solidity Summit and the upcoming Solidity Underhanded Contest as well as exploring all kinds of tools which facilitate a better exchange between the Solidity core-team and the wider Solidity ecosystem. These efforts mostly focus on fostering the language design process, e.g. with language design discussions and updates via the solidity-users forum.*
+
+### What would you like to see happening in the Solidity ecosystem in the future?
+
+*We're building a strong Solidity ecosystem of tool builders, auditors, security researchers, super users and contributors which all regularly contribute to relevant language design discussions. I would love to see this ecosystem steadily grow and hope we find good interaction modes to strengthen the exchange between the different stakeholders. I also hope that we'll manage to constantly evolve and eventually define a transparent and lightweight process for the language design, which allows for collaboration and innovation.*
+
+## Hari - [@hrkrshnn](https://github.com/hrkrshnn)
+
+### What's your professional background:
+*I'm a recent graduate in Math (I like Geometry, Topology, Hyperbolic Geometry, etc.). During my studies, I also used to work on a Linear-Programming solver at the Zuse Institute.*
+
+### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+*I'm a software developer at Solidity. Nowadays, I quite like working on the Yul optimizer. It is interesting since ultimately it's an open problem and allows for a lot of room for improvement.*
```

Comment:
```suggestion
*Nowadays, I quite like working on the Yul optimizer. It is interesting since ultimately it's an open problem and allows for a lot of room for improvement.*
```
I guess I didn't quite understand the context. Please remove that line.